recipes/ncrystal/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x
+set -e
+which cmake
+cmake --version
+$PYTHON --version
+mkdir -p build
+cd build
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    "${SRC_DIR}/src" \
+    -DBUILD_SHARED_LIBS=ON \
+    -DNCRYSTAL_NOTOUCH_CMAKE_BUILD_TYPE=ON \
+    -DMODIFY_RPATH=OFF \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DNCrystal_CMAKEDIR=lib/cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_EXAMPLES=OFF \
+    -DINSTALL_SETUPSH=OFF \
+    -DEMBED_DATA=ON \
+    -DINSTALL_DATA=OFF \
+    -DNO_DIRECT_PYMODINST=ON \
+    -DPython3_EXECUTABLE="$PYTHON" \
+    ${CMAKE_ARGS}
+
+make -j${CPU_COUNT:-1}
+make install
```

Comment:
Is there a `make check` or `ctest` we can run here?

Reply:
There is not, but I have added a comment in the text concerning potentially adding one in the future.